spec/jobs/update_plage_ouvertures_expirations_job_spec.rb

```diff
@@ -0,0 +1,24 @@
+RSpec.describe UpdatePlageOuverturesExpirationsJob, type: :job do
+  subject do
+    UpdatePlageOuverturesExpirationsJob.perform_now
+  end
+
+  context "when plage ouverture are expired" do
+    let!(:plage_ouverture_reguliere) { create(:plage_ouverture, recurrence: Montrose.every(:week, until: Time.now).to_json) }
+    let!(:plage_ouverture_exceptionnelle) { create(:plage_ouverture, :no_recurrence, first_day: Date.today) }
+
+    it "should call notification service" do
+      travel_to(2.days.from_now)
+      expect { subject }.to change(PlageOuverture.where(expired_cached: true), :count).from(0).to(2)
+    end
+  end
+
+  context "when plage ouverture are not expired" do
+    let!(:plage_ouverture_reguliere) { create(:plage_ouverture, recurrence: Montrose.every(:week, until: Time.now).to_json) }
+    let!(:plage_ouverture_exceptionnelle) { create(:plage_ouverture, :no_recurrence, first_day: Date.today) }
```

Comment:
imho you could move the common plage ouvertures creation up in the parent `describe` block since they're exactly the same and this file is small so it's easy not to lose context